doc/articles/uno-development/debugging-uno-ui.md

```diff
@@ -24,23 +24,55 @@ See [this article](working-with-the-samples-apps.md) for more information on wor
 
 It is also possible to debug Uno.UI code in an application outside the Uno.UI solution. The Uno.UI build process has an opt-in mechanism to overwrite the contents of the NuGet cache, causing the application to use your local build of Uno.
 
+> [!NOTE]
+> This will ***overwrite your local NuGet cache***.
+
 This is useful when debugging a problem that can't easily be reproduced outside the context of the app where it was discovered.
 
 It can even speed up your development loop when working on a new feature or fixing a bug with a standalone repro, because a small 'Hello World' app builds considerably faster than the full SamplesApp.
 
-First, you'll need to install, in the app you want to debug with, a published Uno.WinUI package version which is close to your branch's code, preferably a `-dev.xxx` version. When using an Uno.SDK app, this version can be found in the readme file of the Uno.Sdk package, or in the solution explorer packages node for the project.
+First, update `global.json` of the app you will be debugging to instead use a `-dev.xxx` version:
+
+```diff
+diff --git a/global.json b/global.json
+index 509ebab..a2bb2f7 100644
+--- a/global.json
++++ b/global.json
+@@ -1,9 +1,9 @@
+ {
+   // To update the version of Uno please update the version of the Uno.Sdk here. See https://aka.platform.uno/upgrade-uno-packages for more information.
+   "msbuild-sdks": {
+-    "Uno.Sdk": "6.1.23"
++    "Uno.Sdk": "6.2.0-dev.59"
+   },
```

Comment:
dont need the full diff here, this should be enough
```suggestion
 {
   // To update the version of Uno please update the version of the Uno.Sdk here. See https://aka.platform.uno/upgrade-uno-packages for more information.
   "msbuild-sdks": {
-    "Uno.Sdk": "6.1.23"
+    "Uno.Sdk": "6.2.0-dev.59"
   },
```
we mentioned about the file name in previous line, so that is redundant too